<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>esiCancer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Simulation Parameters</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Seed</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="seedSpin">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>13</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Number of Cells</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="numberOfCellsSpin">
         <property name="maximum">
          <number>99999999</number>
         </property>
         <property name="singleStep">
          <number>100</number>
         </property>
         <property name="value">
          <number>1000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QPushButton" name="startSimulationButton">
         <property name="text">
          <string>Start Simulation</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="forceStopButton">
         <property name="text">
          <string>Force Stop</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Standard Values</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Prob of Division</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Prob of Death</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Max Divisions</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="probProlSpin">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.000100000000000</double>
         </property>
         <property name="value">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="probDeathSpin">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.000100000000000</double>
         </property>
         <property name="value">
          <double>0.010000000000000</double>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxDivisionsSpin">
         <property name="maximum">
          <number>9999</number>
         </property>
         <property name="value">
          <number>30</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="mutationsPerDivisionSpin">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>20</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Events Per Division</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="LoadTableButton">
         <property name="text">
          <string>Load esiTable</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="LoadSinergyButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Load Interaction Table</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame_4">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Automatic Runs</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Parameter to Iterate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBoxParameterToIterate">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frame">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Seed</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Min Value</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Max Value</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Increment</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <widget class="QPushButton" name="runAutomaticsButton">
         <property name="text">
          <string>Run Automatics</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="minValueAutomaticSpin">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxValueAutomaticSpin">
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="incrementAutomaticSpin_2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="QProgressBar" name="automaticsProgressBar">
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_19">
         <property name="text">
          <string>Output files</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBoxOutputFiles">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>for each Seed</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>for entire Automatics</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>both</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2" colspan="2">
     <widget class="QFrame" name="frame_ManualMutations">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Manual Events</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableMutationsManual">
         <column>
          <property name="text">
           <string>Generation</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Percentage</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Gene-Event Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Alleles</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="addNewRuleButton">
            <property name="text">
             <string>Add New Rule</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="editRuleButton">
            <property name="text">
             <string>Edit Rule</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="deleteRuleButton">
            <property name="text">
             <string>Delete Rule</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="clearAllMutationManual">
            <property name="text">
             <string>Clear All</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QFrame" name="frame_3">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Stop Conditions</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Number of Generations (-1 for infinite)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="numberOfGenerationsStopSpin">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>1101</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Number of Cells  (-1 for infinite)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="numberOfCellsStopSpin">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>999999999</number>
         </property>
         <property name="value">
          <number>10000</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Affected Cells  (-1 for infinite)</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="mutatedCellsStopSpin">
         <property name="minimum">
          <number>-1</number>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
         <property name="value">
          <number>-1</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QFrame" name="frame_5">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>9</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Tumor Environment</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="stdGrowthRateCheckBox">
         <property name="text">
          <string>Enable</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="stdGrowthRateLabel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Maximal Tumor Growth Rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDoubleSpinBox" name="stdGrowthRateSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>2.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
